main.js

```diff
@@ -1,53 +1,176 @@
 /* eslint-disable no-unused-vars */
-
 var products = [
   {
     image: 'images/turquoise.jpg',
-    title: 'Something With Turquoise'
+    title: 'Something With Turquoise',
+    description: 'This original signed photograph is professionally printed at a lab on archival photography paper.',
+    size:
+    { small: 'Small: 8x8 inches Unframed Print (approx 20.3cm sq.)',
```

Comment:
The opening curly brace for object literals should be a separate line from the first property.
```
{
  small: 'Small...
}
```